bin/lambda-cfn-deploy.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
We can delete this entire `bin/lambda-cfn-deploy.js` file now 🔪  as people should now use `bin/lambda-cfn.js` instead 🎉 .